src/screens/TreeDetailsScreen.tsx

```diff
@@ -83,6 +84,10 @@ export default function TreeDetailsScreen({
         style={styles.input}
         value={entry.id}
       />
+      <Button
```

Comment:
Very confusing that this is a button & clickable when not in edit mode -- i was clicking on it expecting it to save to firebase. Ik this is all temporary but for the sake of bug bash, change to regular text like Status: Active